### DISCUSSION TITLE
GHCR login denied

### DISCUSSION THREAD

**Discussion by ltm** [community/community#26252] [Actions]
<p>Within the past few hours GHCR has started returning “denied” when attempting to log in:</p>
<pre><code class="lang-nohighlight">$ echo $PAT | docker login ghcr.io -u ltm --password-stdin
Error response from daemon: Get https://ghcr.io/v2/: denied: denied
</code></pre>
<p>This happens with both an existing token that used to work and a brand new token. There are no current issues listed on <a href="http://githubstatus.com" rel="noopener nofollow ugc">githubstatus.com</a>. Are others affected by this?</p>

**Comment by shayneburgess** [comment, marked as answer]
<p>Lars, thanks for reporting this. We believe this should be working now - can you verify if you are still seeing this error?</p>

**Comment by ltm** [comment]
<p><a class="mention" href="/shayneburgess">@shayneburgess</a> Thanks! It’s working now.</p>

**Comment by gabr1elt-arc** [comment]
<p>I seem to be finding the same problem on Windows cmd.exe and WSL2.</p>
<pre><code class="lang-auto">echo $PAT | docker login ghcr.io -u gabr1elt-arc --password-stdin
Error response from daemon: Get https://ghcr.io/v2/: denied
</code></pre>
<p>Just tested it on Mac and It works fine there.</p>

**Comment by VinodKumar002** [comment]
<p>I am facing same issue<br>
double checked my PAT has all access</p>
<p>docker login <a href="https://ghcr.io" rel="noopener nofollow ugc">https://ghcr.io</a> -u  -p $CR_TOKEN<br>
WARNING! Using --password via the CLI is insecure. Use --password-stdin.<br>
Error response from daemon: <em>Get <a href="https://ghcr.io/v2/:" rel="noopener nofollow ugc">https://ghcr.io/v2/:</a> denied</em></p>

**Comment by gabr1elt-arc** [comment]
<p>It appears to have broken again!!!<br>
Any clues on why this happens?</p>

**Comment by cbonami** [comment]
<p>Still broken. Strange thing is that I can use the PAT in Github Actions, but when I use my sheel, it doesn’t work.</p>

**Comment by mchaviaras** [comment]
<p>Same problem for me. I regenerate the token and the error disappeared.</p>

**Comment by korsmakolnikov** [comment]
<p>same problem</p>
<p>but this works <a href="https://github.community/t/ghcr-login-denied/142467/5" class="inline-onebox">GHCR login denied - #5 by VinodKumar002</a></p>

**Comment by mongkonwirunputi** [comment]
<p>This error has been occur on windows but macOs is works.</p>

**Comment by mongkonwirunputi** [comment]
<p>It’s broken again for windows but works on macOS</p>

**Comment by Gorgeous-Patrick** [comment]
It is broken again on my MacOS

**Comment by Kamikaze01git** [comment]
Same problem with portainer

"https://ghcr.io/v2/...: denied: denied

**Comment by terion-name** [comment]
mac os - not working

**Comment by caviri** [comment]
For those like me please pay attention to typos. 

**It's `ghcr.io` not `ghrc.io`**
